### PR TITLE
ScoutDAO

### PR DESCRIPTION
**Pull request by burrrata**
Create ScoutDAO to help the Aragon community validate ideas, get more devs working on the Aragon stack, and increase the value of Aragon as a platform.

### CODE REVIEW

File: AGPs/AGP-ScoutDAO.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Status: Stage III
+Track: Finance
+Created: 2019-10-02
+---
+
+# AGP-X: ScoutDAO
+
+## Address of the transfer recipient
+
+ScoutDAO (DAO vault address)
```
Comment (izqui) [inline AGPs/AGP-ScoutDAO.md]: Please input the address of the DAO or the one person or entity that will provide the address

File: AGPs/AGP-ScoutDAO.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## Amount of the transfer
+
+250k DAI
+50k ANT
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+1, but maybe more if the amount of successful projects is higher than expectations.
```
Comment (izqui) [inline AGPs/AGP-ScoutDAO.md]:
```suggestion
1
```
Another AGP would need to be made, I don't think that comment belongs there

**Comment by john-light** [comment]
The time now is after the published deadline of October 10, 2019 at 16:00 UTC for finalizing proposals and moving them to Stage V, and this proposal has not yet been finalized. Per [AGP-1](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-1.md#stage-iv-community-review), this proposal is considered to be withdrawn and this pull request will now be closed:

> If you do not finalize the proposal and request to move the proposal to Stage V before Stage V is scheduled to begin, AGP Editors will consider the proposal withdrawn and close the pull request when Stage V begins.

**Comment by john-light** [comment]
The Aragon Association Board of Directors has voted to push back the ANV-4 schedule by one week. This PR is being re-opened and is included in the Community Review Period which will now last until October 17, 2019 at 16:00 UTC. Please finalize the draft and leave a comment saying "This draft is final and ready to move to Stage V" before the end of the Community Review Period if you would like this proposal to be reviewed by the Aragon Association in Stage V.

Ref: https://forum.aragon.org/t/anv-4-delay-announcement/1367

**Comment by burrrata** [comment]
As per [this discussion on the Aragon Forum](https://forum.aragon.org/t/agp-discussion-scoutdao/1331) the ScoutDAO AGP is being withdrawn. Thank you for the feedback.

**Pull request closed without merging**